partials/functionCalls.vue

```diff
@@ -0,0 +1,269 @@
+            >
+              <FormatAeUnit :value="call.internalTx.amount" />
+            </AppDefinition>
+            <AppDefinition
+              v-if="call.internalTx.queryFee"
+              title="Query Fee"
+            >
+              <FormatAeUnit :value="call.internalTx.queryFee" />
+            </AppDefinition>
+          </AppTableCell>
+        </AppTableRow>
+        <AppTableRow
+          v-if="call.internalTx.queryId"
+          extend
+        >
+          <AppTableCell extend>
+            <AppDefinition
+              type="list"
+              title="Query ID"
+            >
+              {{ call.internalTx.queryId }}
+            </AppDefinition>
+          </AppTableCell>
+        </AppTableRow>
+        <AppTableRow
+          v-if="call.internalTx.query"
+          extend
+        >
+          <AppTableCell extend>
+            <AppDefinition
+              type="list"
+              title="Query"
+            >
+              {{ call.internalTx.query }}
+            </AppDefinition>
+          </AppTableCell>
+        </AppTableRow>
+        <AppTableRow
+          v-if="call.internalTx.queryTtl"
+          extend
+        >
+          <AppTableCell extend>
+            <AppDefinition
+              type="list"
+              title="Query Total"
+            >
+              {{ call.internalTx.queryTtl.type }} : {{ call.internalTx.queryTtl.value }}
+            </AppDefinition>
+          </AppTableCell>
+        </AppTableRow>
+        <AppTableRow
+          v-if="call.internalTx.response"
+          extend
+        >
+          <AppTableCell extend>
+            <AppDefinition
+              type="list"
+              title="Response"
+            >
+              {{ decode(call.internalTx.response.slice(1)) }}
+            </AppDefinition>
+          </AppTableCell>
+        </AppTableRow>
+        <AppTableRow
+          v-if="call.internalTx.responseTtl"
+          extend
+        >
+          <AppTableCell extend>
+            <AppDefinition
+              type="list"
+              title="Response Total"
+            >
+              {{ call.internalTx.responseTtl.type }} : {{ call.internalTx.responseTtl.value }}
+            </AppDefinition>
+          </AppTableCell>
+        </AppTableRow>
+        <AppTableRow
+          v-if="call.internalTx.name && call.internalTx.nameId"
+          extend
+        >
+          <AppTableCell extend>
+            <AppDefinition
+              type="list"
+              title="Name"
+            >
+              <nuxt-link :to="`/names/${call.internalTx.nameId}`">
+                <span class="name">{{ call.internalTx.name }}</span>
+              </nuxt-link>
+            </AppDefinition>
+          </AppTableCell>
+        </AppTableRow>
+        <AppTableRow
+          v-if="call.internalTx.hasOwnProperty('nameTtl')"
+          extend
+        >
+          <AppTableCell extend>
+            <AppDefinition
+              type="list"
+              title="Name Total"
```

Comment:
it means "time to live" (amount in blocks)

```suggestion
              title="Name TTL"
```